src/debugger/daceDebugger.ts

```diff
@@ -0,0 +1,154 @@
+// Copyright 2020-2021 ETH Zurich and the DaCe-VSCode authors.
+// All rights reserved.
+
+import * as vscode from 'vscode';
+import { DaceDebugSession } from './DaceDebugSession';
+import { DaCeInterface } from '../daceInterface'
+import * as os from 'os';
+
+export function activateDaceDebug(context: vscode.ExtensionContext) {
+
+    context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider(
+        'dace-debug',
+        new DaceDebugConfigProvider()
+    ));
+
+    context.subscriptions.push(
+        vscode.debug.registerDebugAdapterDescriptorFactory(
+            'dace-debug',
+            new DaceInlineFactory()
+        )
+    );
+}
+
+class DaceDebugConfigProvider
+implements vscode.DebugConfigurationProvider {
+
+    resolveDebugConfiguration(
+        folder: vscode.WorkspaceFolder | undefined,
+        config: vscode.DebugConfiguration,
+        token?: vscode.CancellationToken
+    ): vscode.ProviderResult<vscode.DebugConfiguration> {
+
+        // if launch.json is missing or empty
```

Comment:
I think you should run an auto formatter on the file

Reply:
Agreed - on this note; I am working on a rule-file for `prettier` which will be added shortly to take care of that. For now, just try to match the remainder of the project. Most notably for this file that means a line length of no more than 80 columns, and (4) space indentation (no tabs). Thanks!

Reply:
I ran a formatter on the file and made sure to not surpass the 80 column line in my latest commit.

Reply:
LGTM, thanks!